test_project.py

```diff
@@ -108,6 +108,12 @@ def test_area_code_look_csv(self):
         results = area_code_lookup(','.join(data['Phone']))
         # make sure they are equal
         self.assertDictEqual(expected, results)
+    
+    def test_area_code_prefixes(self):
+        #test for none U.S. states and territories
+        expected = [487, 522, 652, 642, 784, 591, 880, 348, 933, 987, 627, 893, 595, 866, 211, 627, 328, 766, 969, 544, 995, 492, 421, 381, 249, 594, 482, 494, 596, 889, 932, 411, 696, 789, 922, 900, 902, 358, 871, 388, 550]
+        results = area_code_prefixes()
```

Comment:
Because you implemented area_code_prefixes() without any parameters it makes very difficult to add any negative unit test cases.  A way around this would have been to read the data prior to calling this function and passing that in as a parameter.  That would allow for better control of your data during unit testing and allow you to pass invalid content, empty strings, etc.